src/App.jsx

```diff
@@ -1,8 +1,196 @@
+                                <img src={checkImage} alt="Check Image" />
+                                {option.name}
+                            </div>
+                        );
+                    default:
+                        return null; // Return null for unhandled options
+                }
+            } else {
+                return (
+                    <div key={`${option.name}-${allTV[array].type}`}>
+                        <img src={minusImage} alt="Minus Image" />
+                         {option.name.toLowerCase()}
+                    </div>
+                );
+            }
+        });
+
+        return (
+            <div className={"options-class"}>
+                {availableOptions}
+            </div>
+        );
+    }
+
+
+    return (
+      <>
+      <h1>Verkoopoverzicht tv!</h1>
+          <div className={"parentContainer"}>
+              <div className={"green-container"}>
+                  <h2>Numbers Sold</h2>
+                  <p>{amountOfTvSold()}</p>
+              </div>
+              <div className={"blue-container"}>
+                  <h2>Numbers Stocked</h2>
+                  <p>{amountOfTVBought()}</p>
+              </div>
+              <div className={"orange-container"}>
+                  <h2>Numbers to sale</h2>
+                  <p>{amountToBeSold()}</p>
+              </div>
+          </div>
+
+      <h2>Best selling tv</h2>
+          <div className={"main-container"}>
+              <div className={"product-picture"}>
+                  <img src={tvImage(0)} alt="image-of-a-tv" className={"picture-size"}/>
+              </div>
+              <div className={"product-information"}>
+                  <h3>{tvName(0)}</h3>
+                  <p>{priceOfTV(0)}</p>
+                  <p>{tvSize(0)}</p>
+                  <div>{checkAndMinus(0)}</div>
+              </div>
+          </div>
+          <div>
+              <h2>Alle tv's</h2>
+              <div className={"button-container"}>
+                  <button type= "button" className={"button-class"} onClick={() => meestVerkocht("")}>Meest verkocht eerst</button>
+                  <button type= "button" className={"button-class"} onClick={() => goedkoopste("")}>Goedkoopste eerst</button>
+                  <button type= "button" className={"button-class"} onClick={() => sport("")}>meest geschikt voor sport</button>
+              </div>
+              <div className={"main-container"}>
+                  <div className={"product-picture"}>
+                      <img src={tvImage(1)} alt="image-of-a-tv" className={"picture-size"}/>
+                  </div>
+                  <div className={"product-information"}>
+                      <h3>{tvName(1)}</h3>
+                      <p>{priceOfTV(1)}</p>
+                      <p>{tvSize(1)}</p>
+                      <div>{checkAndMinus(1)}</div>
+                  </div>
+              </div>
```

Comment:
Dit had je in een `map` kunnen doen. Eigenlijk wil je voor elke entry in de array dit stukje doen. Nu wordt er veel code herhaald en is het niet schaalbaar als de array groter/kleiner wordt